pkg/morph/deploy/deploy.go

```diff
@@ -462,7 +440,53 @@ func Deploy(ctx context.Context, prm Prm) error {
 
 	prm.Logger.Info("Audit contract successfully synchronized", zap.Stringer("address", auditContractAddress))
```

Comment:
You still deploy audit the first (from regular non-NNS contracts), but it's named `02-audit` now. Probably a rename or reorder is needed.

Reply:
Audit contract is deployed after the Proxy one

Reply:
OK, then the numbering here and the numbering in contracts don't match exactly. Not a big deal though.